### PR TITLE
Create blogpost to announce admin analytics

### PR DESCRIPTION
**Pull request by iskyOS**
Creating the blog post to announce our new admin analytics tooling.

### CODE REVIEW

File: content/blogposts/2022/admin-analytics.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    url: https://twitter.com/robrhyne
+publishDate: 2022-07-27T18:00+02:00
+tags: [blog]
+slug: admin-analytics
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/admin-analytics/Admin_Analytics_Blog.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/admin-analytics/Admin_Analytics_Blog.png
+published: true
+---
+
+Sourcegraph was created to improve developer efficiency. By providing the ability to see how frequently an API is used across a company’s code, by making it incredibly fast to find and share examples of how an API is used, or understand the scope of a vulnerability and fix it, we’re reducing duplicated efforts, reducing time spent looking for knowledge, and generally making development more efficient.
```
Comment (rrhyne) [inline content/blogposts/2022/admin-analytics.md:15]:
```suggestion
Sourcegraph was created to improve developer efficiency. By providing the ability to see how frequently an API is used across a company’s code, by making it incredibly fast to find and share examples of how a function is used, or understand the scope of a vulnerability and fix it, we’re reducing duplicated efforts, reducing time spent looking for knowledge, and generally making development more efficient. 
```

File: content/blogposts/2022/admin-analytics.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+tags: [blog]
+slug: admin-analytics
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/admin-analytics/Admin_Analytics_Blog.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/admin-analytics/Admin_Analytics_Blog.png
+published: true
+---
+
+Sourcegraph was created to improve developer efficiency. By providing the ability to see how frequently an API is used across a company’s code, by making it incredibly fast to find and share examples of how an API is used, or understand the scope of a vulnerability and fix it, we’re reducing duplicated efforts, reducing time spent looking for knowledge, and generally making development more efficient. 
+
+Delivering that efficiency to end users requires talented and dedicated admins who also deserve great tools. In recognition, we’ve recently taken on the goal of building a world-class product admin experience and we’re excited to release improved admin analytics to help admins understand the value of Sourcegraph.
```
Comment (rrhyne) [inline content/blogposts/2022/admin-analytics.md:17]:
```suggestion
Delivering that efficiency to end users requires talented and dedicated admins who also deserve great tools. In recognition, we’ve recently taken on the goal of building a world-class product admin experience and we’re excited to release improved admin analytics as the first step towards that goal. 
```

**Review by rrhyne — APPROVED** [review]
LGTM

**Review by ryphil — APPROVED** [review]

**Review by rrhyne — APPROVED** [review]
LGTM!

**Review by rrhyne — APPROVED** [review]
STAMP

**Pull request merged**